package.json

```diff
@@ -62,10 +62,10 @@
     "@nestjs/schematics": "^7.0.0",
     "@nestjs/testing": "^7.0.0",
     "@ngrx/schematics": "9.1.0",
-    "@ngrx/store-devtools": "9.1.0",
+    "@ngrx/store-devtools": "^9.1.0",
     "@nrwl/jest": "10.0.12",
-    "@nrwl/nest": "10.0.12",
-    "@nrwl/node": "10.0.12",
+    "@nrwl/nest": "^10.1.0",
```

Comment:
What's happened here?

Are you using the right version of node/npm?

Reply:
Reverted back package json file